scripts/build-windows/build_kclvm_cli.bat

```diff
@@ -0,0 +1,30 @@
+:: Copyright 2021 The KCL Authors. All rights reserved.
+
+setlocal
+cd %~dp0
+
+:: install kclvm-runtime
```

Comment:
The kclvm-runtime needs not to build separately. Ref: https://github.com/KusionStack/KCLVM/pull/343

Reply:
fixed.